TESTING.md

```diff
@@ -176,6 +176,12 @@ Prebuilt development images can be downloaded from here:
 
 Install the package `qemu-system-x86_64` if you don't have already installed.
 
+
+> [!WARNING]
+> Since the commit https://github.com/libremesh/lime-packages/commit/64b7c01961c6e7bc776f83523aea9264724b3b23 lime packages breaks the qemu image and the interfaces cannot be setted up!
```

Comment:
```suggestion
> Since the commit https://github.com/libremesh/lime-packages/commit/64b7c01961c6e7bc776f83523aea9264724b3b23 lime packages break the qemu image and the interfaces cannot be setted up!
```